packages/suite/src/support/messages.ts

```diff
@@ -5000,6 +5005,14 @@ export default defineMessages({
         defaultMessage:
             'Access the NFTs stored in your wallet. Currently available for EVM-based chains only.',
     },
+    TR_EXPERIMENTAL_ETHEREUM_L2_SUPPORT: {
+        id: 'TR_EXPERIMENTAL_ETHEREUM_L2_SUPPORT',
+        defaultMessage: 'Support for Ethereum L2 networks (Optimism, Arbitrum One, Base)',
```

Comment:
```suggestion
        defaultMessage: 'Support for Ethereum L2 networks (Base, Optimism, Arbitrum One)',
```

Reply:
Why like that? If the order matters I'd put it alphabetically (Arbitrum One, Base, Optimism)

Reply:
Base is probably most popular but yea lets stick to order in networks config